forge-gui/res/cardsfolder/upcoming/king_darien_xlviii.txt

```diff
@@ -0,0 +1,11 @@
+Name:King Darien XLVIII
+ManaCost:1 G W
+Types:Legendary Creature Human Soldier
+PT:2/3
+S:Mode$ Continuous | Affected$ Creature.Other+YouCtrl | AddPower$ 1 | AddToughness$ 1 | Description$ Other creatures you control get +1/+1.
+A:AB$ PutCounter | Cost$ 3 G W | CounterType$ P1P1 | CounterNum$ 1 | SubAbility$ DBToken | SpellDescription$ Put a +1/+1 counter on NICKNAME and create a 1/1 white Soldier creature token.
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ w_1_1_soldier | TokenOwner$ You | StackDescription$ None
+A:AB$ PumpAll | Cost$ Sac<1/CARDNAME> | ValidCards$ Creature.token+YouCtrl | KW$ Hexproof & Indestructible | SpellDescription$ Creature tokens you control gain hexproof and indestructible until end of turn.
+DeckHas:Ability$Token & Type$Soldier
+SVar:PlayMain1:TRUE
+Oracle:Other creatures you control get +1/+1.\n{3}{G}{W}: Put a +1/+1 counter on King Darien and create a 1/1 white Soldier creature token.\nSacrifice King Darien: Creature tokens you control gain hexproof and indestructible until end of turn.
```

Comment:
DeckHas:Ability$Counters|Token|Sacrifice
DeckHints:Ability$Token (because of the big token payoff)

Reply:
Extended `DeckHas` params and `DeckHints` param as suggested. Thanks!